docs/social_login.rst

```diff
@@ -111,8 +111,47 @@ prompted to accept the permissions requested:
 After accepting permissions, you'll be immediately redirected back to your
 website at the URL specified by ``redirectUrl`` in your app's config.
 
+.. note::
+
+  By default the configuration ``web.social.facebook.scope`` is set to ``email``. Change
+  this configuration value to request additional scopes.
+
 Simple, right?!
 
+Logging in using an Access Token
```

Comment:
This looks good, but I see what you mean about the title & placement of the new section. I think this section and ‘Test it Out’ should be renamed. I would rename ‘Test it Out’ to something like Standard auth, or Default auth. This implies that the standard/easy way of doing things is to visit /login, click on the FB button, input credentials, etc. Then you can rename your section to something like Facebook SDK Auth, or Facebook Access Token Auth, since this section shows an alternative. Same with Google/LinkedIn sections